networkx/readwrite/json_graph/tree.py

```diff
@@ -5,7 +5,7 @@
 
 
 # NOTE: Remove attrs from signature in 3.0
-def tree_data(G, root, attrs=None, ident="id", children="children"):
+def tree_data(G, root, attrs=None, ident="id", children="children", attr="attr"):
```

Comment:
Is the new kwarg necessary? Can't the attribute names just be grabbed from the `.graph` attribute directly? It should still be possible to check that there are no conflicts with "protected" names by doing something like:

```python
# Check no name collisions between graph attrs and special json keys
if {ident, children} & G.graph.keys():
    raise ValueError
# Check no name collisions between graph attrs and node names
if G.graph.keys() & G.nodes():
    raise ValueError
```